src/components/Dropdown/Dropdown.tsx

```diff
@@ -179,18 +168,18 @@ const DropdownComponent: FC<DropdownProps> = ({
     onNavigate: setActiveIndex,
   });
 
-  const typeahead = useTypeahead(context, {
-    listRef: labelsRef,
-    activeIndex,
-    selectedIndex,
-    onMatch: handleTypeaheadMatch,
-  });
+  // const typeahead = useTypeahead(context, {
```

Comment:
Should not this be removed? Or maybe have a comment above it describing why this needs to be commented but kept